app/Plugin/CoreApi/Model/CoreApi.php

```diff
@@ -113,8 +131,36 @@ class CoreApi extends AppModel {
         'allowEmpty' => false
       )
     ),
+    'index_response_type' => array(
+      'content' => array(
+        'rule' => array('inList', array(ResponseTypeEnum::Full,
+                                        ResponseTypeEnum::IdentifierList)),
+        'required' => true,
+        'allowEmpty' => false
+      )
+    ),
   );
-  
+
+  /**
+   * Actions to take before a validate operation is executed.
+   *
+   * @since  COmanage Registry v4.1.0
+   */
+
+  public function beforeValidate($options = array())
+  {
+    if(empty($this->data['CoreApi']["api"])) {
+      return false;
+    }
+    // Disable Response type for Write/Delete actions
+    if($this->data['CoreApi']["api"] === CoreApiEnum::CoPersonWrite) {
```

Comment:
The Write API implies access to the Read API, so I think we still want the `index_response_type` setting.

Reply:
Thank you for the clarification on that!

Reply:
Done